.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,29 @@
+name: deploy-test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+    - main
+
+jobs:
+  deploy-basic:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-go@v4
+      with:
+        go-version-file: go.mod
+
+    - name: Run basic unit tests
+      run: |
+        make kind-cluster
+        make deploy
+        kubectl get crds operators.operators.operatorframework.io
+        kubectl get ns operator-controller-system
+        make undeploy
+        ! kubectl get ns operator-controller-system
+        ! kubectl get crds operators.operators.operatorframework.io
```

Comment:
Two things:
1. nit: it looks like there's a mix of spaces and tabs here for indentation (unless GH UI is tricking me). Change tabs to spaces (only because that's how the rest of the file looks, not trying to start a holy war 😛 )?
2. My last comment I think was eaten by a new commit, repeating here: These steps essentially happen during the course of running the e2e, don't they? I suppose we don't try uninstall and undeploy during the e2e, but maybe we could?

Reply:
Since removing `install`/`uninstall`, I put in `undeploy`

Reply:
There are no tabs... so GH UI must be tricking you?